openreview/conference/builder.py

```diff
@@ -1012,7 +1012,7 @@ def setup_first_deadline_stage(self, force=False, hide_fields=[], allow_author_r
             additional_fields=self.submission_stage.additional_fields,
             remove_fields=self.submission_stage.remove_fields,
             only_accepted=False,
-            multiReply=False,
+            multiReply=False if self.submission_stage.double_blind else True,
```

Comment:
the change looks good, you can add a test in a single blind submission with a second deadline. I think ESWC has that.

Reply:
@melisabok what exactly needs to be tested here?

Reply:
the Revision invitation has the multireply property in True.